media/css/mozorg/mpl-2-0.scss

```diff
@@ -0,0 +1,63 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../pebbles/includes/lib';
+
+body {
```

Comment:
I know this was all just copied over but there's some silly/redundant styling here. Is it worth tidying it up or should we just leave it alone? I might even be inclined to remove the CSS entirely and just let browser defaults handle it.